python/tchannel/handler.py

```diff
@@ -92,8 +94,20 @@ def handle_pre_call(self, message_id, message, connection):
         :param message_id: message id
         :param message: CallRequestMessage or CallRequestContinueMessage
         :param connection: tornado connection
+
+        :except InvalidChecksumException:
+            Raise InvalidChecksumException when the checksum in the message
+            is wrong.
```

Comment:
The docstring is wrong. This method never raises that exception. It actually catches it.